src/apps/btc/btc_common.h

```diff
@@ -31,6 +31,7 @@
 
 #define BTC_PURPOSE_P2WPKH_P2SH (49 + BIP32_INITIAL_HARDENED_CHILD)
 #define BTC_PURPOSE_P2WPKH (84 + BIP32_INITIAL_HARDENED_CHILD)
+#define BTC_PURPOSE_MULTISIG (48 + BIP32_INITIAL_HARDENED_CHILD)
```

Comment:
Why do we define some macros in the header and some in the source file? In my opinion they all should be in the source, since I don't see them being used outside of it.

Reply:
You are totally right, we normally do define them in the source if they are not used elsewhere (like the multisig max signers one ;)). I added a comit to move those. The remaining one will be handled soon.